entries/e/epo.org.json

```
@@ -0,0 +1,16 @@
+{
+  "European Patent Office": {
+    "domain": "epo.org",
+    "tfa": [
+      "sms",
+      "email",
+      "totp"
+    ],
+    "documentation": "https://www.epo.org/en/applying/myepo-services/get-access",
+    "notes": "2FA for all MyEPO Services",
+    "categories": [
+      "legal",
+      "government"
+    ]
```

Since it's a service by the eu for residents of eu countries, a regions array is useful here.
```suggestion
    ],
    "regions": [
      "at",
      "be",
      "bg",
      "hr",
      "cy",
      "cz",
      "dk",
      "ee",
      "fi",
      "fr",
      "de",
      "gr",
      "hu",
      "ie",
      "it",
      "lv",
      "lt",
      "lu",
      "mt",
      "nl",
      "pl",
      "pt",
      "to",
      "sk",
      "si",
      "es",
      "se"
    ]
```

---

In fact, the service is not only used by people from the EU but by patent applicants and lawyers worldwide. It is therefore not an offer that is only used by people from the proposed countries, even if this applies to a large proportion. I would therefore suggest not including the list of countries. If a list should still be included, then I would expand the list, as it is not an authority of the European Union, but has many other member countries, such as Turkey, Switzerland or Finland.